test/support/transform/booster/licensesGeneratorTest.ts

```diff
@@ -0,0 +1,51 @@
+import { NodeFsLocalProject } from "@atomist/automation-client/project/local/NodeFsLocalProject";
+import { InMemoryProject } from "@atomist/automation-client/project/mem/InMemoryProject";
+import { Project } from "@atomist/automation-client/project/Project";
+import { tempdir } from "shelljs";
+import { expect } from "chai";
+import { resolve } from "path";
+import licensesGenerator from "../../../../lib/support/transform/booster/licensesGenerator";
+import { LICENSES_GENERATOR_PATH } from "../../../../lib/constants";
+
+const licensesGeneratorPath = resolve(LICENSES_GENERATOR_PATH);
+const licensesXmlPath = "src/licenses/licenses.xml";
+const pomContent = `
```

Comment:
Perhaps we should have this at the end of the file like the other tests?

Reply:
Any idea what's the common practice in TS? In Java normally all the constants are in the top, that's why I put them there. Otherwise I could move all of them down.

Reply:
The reason I had put them in the bottom is that that's what I saw that the tests in various Atomist modules were doing.
Not really sure is there is any TS convention

Reply:
OK I'll move them down

Reply:
Cool